backend/requirements.txt

```diff
@@ -41,6 +41,7 @@ mypy-extensions==0.4.3
 packaging==20.9
 pathspec==0.8.1
 pluggy==0.13.1
+promise==2.3
```

Comment:
Out of curiosity, what is this package needed for? I'm okay adding it, just curious.

Reply:
Hi Julian. Did you find out what this package was for? That's the only thing left I believe to fix

Reply:
Hey, I don't think it's necessary. If the tests pass without it FF to remove

Reply:
Ok. Thanks!